tests/unit/helpers/FixMovedQuestionConditionsTest.php

```diff
@@ -0,0 +1,219 @@
+<?php
+
+namespace ls\tests;
+
+class FixMovedQuestionConditions extends BaseTest
+{
+    public static function setUpBeforeClass(): void
+    {
+        parent::setupBeforeClass();
+
+        \Yii::import('application.helpers.common_helper', true);
+
+        // Import default survey
+        $filename = self::$surveysFolder . '/survey_fixMovedQuestionConditionsDefault.lss';
+        self::importSurvey($filename);
+
+        \Yii::app()->setConfig('sid', self::$surveyId);
+    }
+
+    /**
+     * Test fixing just one condition.
+     * This test uses the survey previously imported in setupBeforeClass.
```

Comment:
Maybe more detail in test ? Here : if there are an issue, i'm unsure of what is the needed result.